### PR TITLE
{data}[gfbf/2023a] polars v0.20.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Supersedes #19770 

Versions newer than 0.20.2 requires newer version of Rust than what we have for 2023a

Depends on framework fix in EB 5.x:
* [x] https://github.com/easybuilders/easybuild-framework/pull/4248

once EB 5 is released this PR needs a rebuild of the checksums.

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--force-download"
EB_BRANCH=5.0.x

File: easybuild/easyconfigs/p/polars/polars-0.20.2-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,908 @@
...
+    {'is-terminal-0.4.9.tar.gz': 'cb0889898416213fab133e1d33a0e5858a48177452750691bde3666d0fdbaf8b'},
+    {'itertools-0.10.5.tar.gz': 'b0fd2260e829bddf4cb6ea802289de2f86d6a7a690192fbe91b3f46e0f2c8473'},
+    {'itertools-0.11.0.tar.gz': 'b1c173a5686ce8bfa551b3563d0c2170bf24ca44da99c7ca4bfdab5418c3fe57'},
+    {'itoa-1.0.10.tar.gz': 'b1a46d1a171d865aa5f83f92695765caa047a9b4cbae2cbf37dbd613a793fd4c'},
+    {'itoap-1.0.1.tar.gz': '9028f49264629065d057f340a86acb84867925865f73bbf8d47b4d149a7e88b8'},
+    {'jemalloc-sys-0.5.4+5.3.0-patched.tar.gz': 'ac6c1946e1cea1788cbfde01c993b52a10e2da07f4bac608228d1bed20bfebf2'},
+    {'jemallocator-0.5.4.tar.gz': 'a0de374a9f8e63150e6f5e8a60cc14c668226d7a347d8aee1a45766e3c4dd3bc'},
+    {'jobserver-0.1.27.tar.gz': '8c37f63953c4c63420ed5fd3d6d398c719489b9f872b9fa683262f8edd363c7d'},
+    {'js-sys-0.3.66.tar.gz': 'cee9c64da59eae3b50095c18d3e74f8b73c0b86d2792824ff01bbce68ba229ca'},
+    {'jsonpath_lib-0.3.0.tar.gz': 'ae7f51d58613783097caa5281dc7f45b541975ab6067be49945119dae27cf5b3'},
```
Comment (lexming) [inline easybuild/easyconfigs/p/polars/polars-0.20.2-gfbf-2023a.eb:196]: Due to recent changes on the creation of reproducible archives in `5.0.x`, this checksum has changed:
```suggestion
    {'jsonpath_lib-0.3.0.tar.gz': 'dfb5eb7f47b5b5a7b35262f7b41787a785958001a23ff039cdae926396c6af96'},
```

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--force-download"
EB_BRANCH=5.0.x

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Merging, thanks @akesandgren !

**Comment by boegel** [comment]
I've manually installed this easyconfig on `generoso` using `--ignore-checksums`, it was merged prematurely, since it depends on a new feature implemented in EasyBuild 5.0 (consistent checksums for source tarballs created by cloning a Git repository)

**Comment by smoors** [comment]
> I've manually installed this easyconfig on `generoso` using `--ignore-checksums`, it was merged prematurely, since it depends on a new feature implemented in EasyBuild 5.0 (consistent checksums for source tarballs created by cloning a Git repository)

thanks! looks like generoso doesn't support the `EB_BRANCH` option

**Comment by boegel** [comment]
> > I've manually installed this easyconfig on `generoso` using `--ignore-checksums`, it was merged prematurely, since it depends on a new feature implemented in EasyBuild 5.0 (consistent checksums for source tarballs created by cloning a Git repository)
> 
> thanks! looks like generoso doesn't support the `EB_BRANCH` option

Indeed, that's only on `jsc-zen3`

**Pull request merged**